packages/okta-angular/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.1.0
+
+### Features
+
+-[](TODO: add after merged) - Adds TokenManager configuration parameters.
```

Comment:
Will update after https://github.com/okta/okta-oidc-js/pull/388 is merged

Reply:
Why not update now if you know what will be in there?

Reply:
The SHA of the merged commit in master (see previous CHANGELOG entries), which will be available after we merge #388 .

Reply:
oh, d'uh.   I somehow thought this was README.